<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="en_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>40</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY목각파임B</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(170, 0, 255);background-color:rgb(85, 255, 255);border-color:rgb(0, 0, 0);border-style : solid;border-width : 3px</string>
    </property>
    <property name="text">
     <string>번역</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>18</y>
      <width>56</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY견명조</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>한국어</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ko_text">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 255);background-color:rgb(255, 255, 127);border-color:rgb(255, 85, 0);border-style : solid;border-width : 3px</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="en_text">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>124</y>
      <width>200</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 127);background-color:rgb(255, 255, 127);border-color:rgb(255, 85, 0);border-style : solid;border-width : 3px</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>98</y>
      <width>56</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY견명조</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>영어</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>228</y>
      <width>56</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY견명조</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>영어</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="en_line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 127);background-color:rgb(170, 255, 127);border-color:rgb(85, 85, 255);border-style : solid;border-width : 3px</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ko_btn">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>250</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY목각파임B</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(170, 0, 255);background-color:rgb(85, 255, 255);border-color:rgb(0, 0, 0);border-style : solid;border-width : 3px</string>
    </property>
    <property name="text">
     <string>번역</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>308</y>
      <width>56</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY견명조</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>한국어</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ko_line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>200</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 255);background-color:rgb(170, 255, 127);border-color:rgb(85, 85, 255);border-style : solid;border-width : 3px</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>HY중고딕</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);background-color:rgb(170, 170, 255);border-color:rgb(0, 0, 0);border-style : solid;border-width : 3px</string>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
